### PR TITLE
[ICD] Update ICDM Cluster XML to latest spec

### LINKED ISSUE #30357: [1.3] ICD Management Cluster XML has to be updated as per spec
**Description**
1. XML should remove the attributes which are not available in spec those are
UserActiveModeTriggerHint
UserActiveModeTriggerInstruction

**PFA XML Screenshot :** 
image

**PFA Spec Screenshot** 
image

2. below mentioned command is missed to define in xml
StayActiveResponse

**Spec Reference :** 
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/ICDManagement.adoc

**XML Reference :** 
https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Description 
fixes #30357

### CODE REVIEW

**Review by yunhanw-google — APPROVED** [review]

File: src/controller/data_model/controller-clusters.matter [hunk 1]
```diff
@@ -2827,6 +2827,10 @@ client cluster IcdManagement = 70 {
     optional octet_string<16> verificationKey = 1;
   }
 
+  response struct StayActiveResponse = 4 {
```
Comment (andy31415) [inline src/controller/data_model/controller-clusters.matter:2830]: this does not show up as a parameter to `StayActiveRequest`. Maybe a `response=` is missing in the XML?

File: src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml [hunk 2]
```diff
@@ -90,12 +90,12 @@ limitations under the License.
         </command>
 
         <command source="server" code="0x01" name="RegisterClientResponse" optional="false" disableDefaultResponse="true">
-            <description> RegisterClient response command </description>
+            <description> RegisterClient response command</description>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml:93]: if we remove trailing blank ... how about removing the prefix blank too?

File: src/controller/data_model/controller-clusters.matter (hunk 1 above)
Comment (mkardous-silabs) [reply src/controller/data_model/controller-clusters.matter:2830]: went a bit too fast. updated

File: src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml (hunk 2 above)
Comment (mkardous-silabs) [reply src/app/zap-templates/zcl/data-model/chip/icd-management-cluster.xml]: that wouldn't hurt, done.

**Review by andy31415 — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**